### ISSUE TITLE
Not clear how to handle cross platform development (Windows 10 VM disables add-ins)

### ISSUE THREAD

**Issue by fergiemcdowall** [OfficeDev/office-js-docs#1068]
Somewhat related to https://dev.office.com/docs/add-ins/word/word-add-ins?product=word

I am developing a commercial add-in and want to test it on different platforms. This seems to go ok on mac and office365, but **not on the Windows 10 VM**

When I download VMs from https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/ and install Office with my Office365 subscription, Word seems to have add-ins disabled somehow. I cant load add-ins from the store, and I can't sideload add-ins that work elsewhere. Add-ins show up as available, but get permanently stuck in their "loading" state when invoked.

Long story short: There seems to be some undocumented setting or limitation that is preventing add-ins from functioning normally on the test VMs. It would be great to get some documentation on how to test add-ins in Windows 10 VMs.

(see also Stackoverflow question: https://stackoverflow.com/questions/45976840/cant-load-any-add-in-into-ms-word-on-windows)

**Comment by Rick-Kirkham** [comment]
Thanks for creating the SO question. That is the right way to raise questions like these. The Issues for this repo are intended for reporting errors/bugs in the documentation.
